provider/version/version.go

```diff
@@ -15,10 +15,12 @@
 package version
 
 import (
+	"strings"
+
 	"github.com/blang/semver"
 )
 
-var Version semver.Version = semver.MustParse(version)
+var Version semver.Version = semver.MustParse(strings.TrimPrefix(version, "v"))
```

Comment:
This fixes #791. The problem was that the release linked in a version of `v6.0.0` (not valid semver) where `6.0.0` was expected. Trimming the `"v"` solves the problem.